README.md

```diff
@@ -1 +1,2 @@
 # retail-url-scraper
+A web scraper that scrapes retail websites and returns URLs for all listed products. Runs on Express and Node
```

Comment:
Add some basic instructions on how to setup the repo:
1. npm i
2. npm start....

+ state the versions of node and npm